tests/RRuleTest.php

```diff
@@ -1945,11 +1945,70 @@ public function testDateTimeMutableReferenceBug()
 	}
 
 	/**
-	 * Test that the RRule::humanReadable() does not fail
-	 * This test does any real testing only if Intl hasn't been loaded
-	 * Or setlocale() returns unusable data
+	 * Providing a set of valid locales to call RRule::i18nLoad() with
+	 *
+	 * @return array
 	 */
-	public function testHumanReadable()
+	public function validFallbackLocales()
+	{
+		return array(
+			array('en'),
+			array('fr'),
+			array('en_US'),
+			array('en_US_POSIX'),
+			array('en_US.utf-8'),
+		);
+	}
+
+	/**
+	 * Test that the RRule::i18nLoad() does not fail when provided with valid locales
+	 *
+	 * @dataProvider validFallbackLocales
+	 */
+	public function testHumanReadable($fallback)
+	{
+		$date = date_create('2007-01-01');
+		$rrule = new RRule(array(
+			'freq' => 'daily',
+			'count' => 10,
+			'dtstart' => $date
+		));
+
+		$reflector = new ReflectionClass('RRule\RRule');
+
+		// Force RRule::$intl_loaded to false, to test fallback locales
+		$property = $reflector->getProperty('intl_loaded');
+		$property->setAccessible(true);
+		$property->setValue($rrule, false);
+
+		setlocale(LC_MESSAGES, 'C');
+		$method = $reflector->getMethod('humanReadable');
+		$result = $method->invokeArgs($rrule, array(array('fallback' => $fallback)));
+		$this->assertNotEmpty($result);
+	}
+
+	/**
+	 * Providing a set of invalid locales to call RRule::i18nLoad() with
+	 *
+	 * @return array
+	 */
+	public function invalidFallbackLocales()
+	{
+		return array(
+			array('eng'),
+			array('invalid'),
+			array('it'),
```

Comment:
This will fail once somebody make a PR to add Italian language :) You should use "xx" instead.